### PR TITLE
Updated README

### LINKED ISSUE #2: Decide purpose of this repo and loosely assign features to members
Description:

Will this repo be about sorting? Displaying something? We need to assign which branches everyone will create and merge in, as well as some knowledge about merge order.

### PR DESCRIPTION
**Pull request by EliasPinno**
Closes #2  

Updated readme to reflect project scope.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
-# cosc-499-team-ex
\ No newline at end of file
+# COSC 499 Team 8 Github excercise
+
+We have decided that this repository will be a demonstration of several sorting algorithms for sorting numbers.
+
+We will demonstrate several different sorting algorithms, with each member opening their own PR and demonstrating their own sorting algorithm.
+
+## Task List (Replicated on GitHub)
+Bogo Sort: Keyvan Khademi
```
Comment (IvanIsCoding) [inline README.md]: This one has a different order than the other ones

File: README.md [hunk 2]
```diff
@@ -1 +1,11 @@
-# cosc-499-team-ex
\ No newline at end of file
+# COSC 499 Team 8 Github excercise
+
+We have decided that this repository will be a demonstration of several sorting algorithms for sorting numbers.
+
+We will demonstrate several different sorting algorithms, with each member opening their own PR and demonstrating their own sorting algorithm.
```
Comment (keyvankhademi) [inline README.md:5]: Don't we need to write tests?

File: README.md (hunk 1 above)
Comment (EliasPinno) [reply README.md]: Fixed this small formatting issue. Thanks for checking!

File: README.md (hunk 2 above)
Comment (EliasPinno) [reply README.md:5]: We do, I've added this requirement.

**Review by IvanIsCoding — APPROVED** [review]
LGTM

**Pull request merged**